common/djangoapps/student/cookies.py

```diff
@@ -5,6 +5,8 @@
 import time
 import json
 
+from email_marketing.cookies import add_email_marketing_cookies
```

Comment:
Oh, cool. I hadn't seen the return-values for signals. That seems like it would be better.